### PR TITLE
Relieve the maximum number of sub-items limit for sublayout method

### PR DESCRIPTION
**Pull request by della-padula**
Relieve the maximum number of sub-items limit for sublayout method #249

From: 7
To: 10

sublayout {
      item
      item
      item
      ...
      item
}

I would like to review whether this function works well.
Thank you.

### CODE REVIEW

File: Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift [hunk 1]
```diff
@@ -34,27 +34,39 @@ extension LayoutBuilder {
     public static func buildBlock<L: Layout, L1: Layout>(_ l: L, _ l1: L1) -> TupleLayout {
         TupleLayout(l, l1)
     }
-
+
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: Please remove unnecessary whitespace

Comment (della-padula) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: Originally, there was a whitespace, but it seems to have been caused by the code alignment operation
I opened the file and it seems that the blanking was handled as intended
Please check it again

Comment (oozoofrog) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: @gmlwhdtjd funny, @della-padula is right. may be github did something...?

Comment (oozoofrog) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: currently, every white space only lines has white spaces... 😂

Comment (oozoofrog) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: i do not checked trimming option in xcode settings. starting clean #251

Comment (oozoofrog) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: @della-padula @gmlwhdtjd so trimming "including whitespace-only line" now our convention?

Comment (della-padula) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: @oozoofrog Yes~ I think it would be nice to set it as a convention from now on~

Comment (gmlwhdtjd) [reply Sources/SwiftLayout/Core/Builder/LayoutBuilder.swift:37]: @oozoofrog @della-padula Good!! I totally agree!

I will resolve the conversation as it has nothing to do with this pr.
We'll cover the whitespace-related part in #251

**Review by gmlwhdtjd — APPROVED** [review]
Thank!
